.github/workflows/test.yml

```diff
@@ -106,10 +106,21 @@ jobs:
         if: ${{ needs.tests.result != 'success' }}
         run: exit 1
 
+  fetch-test-durations:
+    runs-on: ubuntu-latest
+    steps:
+      # fetch test durations
+      # NOTE: if the tests get poorly distributed, run this and commit the resulting `.test_durations` file to the `vyper-test-durations` repo.
+      # `TOXENV=fuzzing tox -r -- --store-durations --reruns 10 --reruns-delay 1 -r aR tests/`
+      - name: Fetch test-durations
+        run: curl --location "https://raw.githubusercontent.com/vyperlang/vyper-test-durations/2879eae3fefb4eb664914df93e545f9d15fe74fb/test_durations" -o .test_durations
```

Comment:
This file can't be commited to the repo here?

Reply:
it's just really big, leads to repo bloat.

Reply:
How big is it?

Reply:
100k lines / 17MB